cardano-client/src/Cardano/Client/API2.hs

```diff
@@ -0,0 +1,318 @@
+-- consensus uses byron types
+-- type-instantiations for codecs
+
+module Cardano.Client.API
```

Comment:
The file name and module name are different.

Do you think we should export both modules from `Cardano.Client` module?  It could be convenient to load everything with just a single (qualified) import.

Reply:
This file is just a leftover, I had forgotten to delete.